js/demo-app/src/index.js

```diff
@@ -181,13 +187,17 @@ function buildNavMarkup(navConfigs) {
     navMenu.appendChild(currLi);
 
     // Build Programmatic Button
+    let currLinkButton = document.createElement('li');
+    currLinkButton.setAttribute('class', `nav-id-${curr.id}`);
     let currButton = document.createElement('button');
     currButton.setAttribute('class', `nav-id-${curr.id}`);
     currButton.addEventListener('click', () => {
+      updateActiveNav(NAV_CONFIGS, curr.id);
```

Comment:
D'oh, too much functional!  I guess i should have made the navs the second argument so we only have have to pass the custom route in this case and no params in the other.  Feel free to switch this around if you want.